Lib/ufoLib/glifLib.py

```diff
@@ -441,7 +447,10 @@ def _readPlist(self, path):
 			with open(path, "rb") as f:
 				data = readPlist(f)
 			return data
-		except:
+		except Exception as e:
```

Comment:
What other Exception except for IOError are you trying to catch here? I don’t like catch-all exceptions

Reply:
It was already catch all before, since the library raises GlifLIbError for all exceptions or do you want me to do two except with one that's unnamed?

Reply:
I see.. I don’t like too much that GlifLibError is eating up anything that happens in that block and function calls without even providing the exception error message.

Reply:
Agreed, but that's probably out of scope for this PR.

Reply:
Ok